go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/vocdoni/vocdoni-z-sandbox
 
 go 1.23.2
 
+replace github.com/consensys/gnark => github.com/lucasmenendez/gnark v0.5.2-0.20241217104836-b7c9c2fcbe8c
```

Comment:
This should not be here

Reply:
until this PR is merged, it is required...